zero_bin/tools/prove_rpc.sh

```diff
@@ -40,7 +40,7 @@ fi
 TOOLS_DIR=$(dirname $(realpath "$0"))
 
 # Set the environment variable to let the binary know that we're running in the project workspace.
-export CARGO_WORKSPACE_DIR="${TOOLS_DIR}/../"
+export CARGO_WORKSPACE_DIR="${TOOLS_DIR}/../../"
```

Comment:
shouldn't this be:
```suggestion
export ZK_EVM_WORKSPACE_DIR="${TOOLS_DIR}/../../"
```
?

Reply:
Actually, hold off merging this until #405 is merged (I'll post something below).

Reply:
You mentioned removing it but it seems to still be here?

Reply:
@BGluth I think you missed this 🙈

Reply:
I think I removed this? :eyes:

Reply:
For `prove_stdio.sh` but not `prove_rpc.sh` 👀

Reply:
Ok it's really gone now. :slightly_smiling_face: